0001-Interrim-solution-for-Unicode-support.patch

```diff
@@ -0,0 +1,39 @@
+From 4e2578b1302a098363e6fca7217a9f368b57d727 Mon Sep 17 00:00:00 2001
```

Comment:
Looks good, should this patch be submitted upstream though?

Reply:
An upstream PR would be great!